.github/workflows/review-checks.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     - uses: actions/checkout@v1
     - name: Install utils
       run: |
-        yum install -y git wget
+        yum install -y git wget ca-certificates -y
```

Comment:
:eyes: `-y [...] -y` yum now knows you mean business

Reply:
🤦🏻 Thanks